code/obj/blob.dm

```diff
@@ -1176,10 +1193,11 @@ var/image/blob_icon_cache
 			added = 0
 
 
-/obj/blob/wall
+/obj/blob/wall // TODO: Add check for if theres a wall to the south, if so replace sprite to pseudo-tile
```

Comment:
TODO?

Reply:
oh shit sorry this is left over, ill remove it